smithy-swift-codegen/src/test/kotlin/StructEncodeGenerationIsolatedTests.kt

```diff
@@ -7,7 +7,7 @@ import io.kotest.matchers.string.shouldContainOnlyOnce
 import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Test
 
-class StructEncodeGenerationIsolatedTests {
+class varStructEncodeGenerationIsolatedTests {
```

Comment:
Class name should probably start with a capital letter

Reply:
ha corrected